src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/service/bulkscan/transformation/FormAToCaseTransformer.java

```diff
@@ -39,30 +43,26 @@
     protected Map<String, Object> runFormSpecificTransformation(List<OcrDataField> ocrDataFields) {
         Map<String, Object> transformedCaseData = new HashMap<>();
 
-        mapFullNameToFirstAndLast(OcrFieldName.APPLICANT_FULL_NAME, "applicantFMName", "applicantLName",
-            ocrDataFields, transformedCaseData);
-        mapFullNameToFirstAndLast(OcrFieldName.RESPONDENT_FULL_NAME, "appRespondentFMname", "appRespondentLName",
-            ocrDataFields, transformedCaseData);
+        mapFullNameToFirstAndLast(APPLICANT_FULL_NAME, "applicantFMName", "applicantLName", ocrDataFields, transformedCaseData);
```

Comment:
actually I kept `OcrFieldName` qualifier to distinct between OCRed form field names and CCD field names. Some CCD field names are in `CCDConfigConstant` - we might think about having them all there, we would need to think about Consented / Contensted cases though.